superset/assets/javascripts/dashboard/reducers/allSlices.js

```diff
@@ -0,0 +1,74 @@
+import * as actions from '../actions/allSlices';
```

Comment:
what does `allSlices` mean? if it is for data, please change the name to be more descriptive, `chartData`?